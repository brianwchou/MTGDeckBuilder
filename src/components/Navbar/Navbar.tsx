import React from 'react';
import "./Navbar.css"

export default function Navbar () {
    return (
        <nav>
            <ul className = "nav">
                <li className = "horizontal"><a className = "changeColor" href="home">Home</a></li>
            </ul>
        </nav>
    )
}