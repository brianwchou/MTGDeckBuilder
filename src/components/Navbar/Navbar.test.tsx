import React from 'react';
import Navbar from '../../components/Navbar'
import renderer from 'react-test-renderer'

describe('Navbar', () => {

    test('renders correctly', () => {
        const tree = renderer
            .create(<Navbar />)
            .toJSON();
         
        expect(tree).toMatchSnapshot();    
    })
})