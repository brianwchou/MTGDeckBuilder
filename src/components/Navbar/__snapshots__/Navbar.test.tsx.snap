// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  className="ui inverted segment"
>
  <div
    className="ui inverted secondary menu"
  >
    <a
      className="active item"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="item"
      onClick={[Function]}
    >
      Decks
    </a>
    <a
      className="item"
      onClick={[Function]}
    >
      Friends
    </a>
  </div>
</div>
`;
